bindings/cs/MongoDB.Libmongocrypt/StatusSafeHandle.cs

```diff
@@ -33,6 +33,16 @@ private StatusSafeHandle()
         {
         }
 
+        private StatusSafeHandle(IntPtr ptr)
+            : base(ptr, false)
+        {
+        }
+
+        public static StatusSafeHandle FromIntPtr(IntPtr ptr)
```

Comment:
I don't see this method usages in the repo. Is it involved in native library functioning?

Reply:
explained above